### PR TITLE
OpenPypeV3: Add key parent asset to path templating construction

### PR DESCRIPTION
**Pull request by ClementHector**
A simple proposal to add the parent in the  path templating construction.

### CODE REVIEW

**Comment by iLLiCiTiT** [comment]
This is much better than adding `parents` I would suggest wait for task types as it will need changes at same places (mostly)?

**Comment by antirotor** [comment]
Blocked by #2157

**Comment by ClementHector** [comment]
Link to [Add parent in template_data_from_session](https://github.com/pypeclub/avalon-core/pull/392)

**Comment by ClementHector** [comment]
@iLLiCiTiT Hound is blocking but I don't know why, can you watch?

**Comment by iLLiCiTiT** [comment]
> Hound is blocking but I don't know why, can you watch

@mkolar interesting hound is houding because of code which is not added, modified or affected by this PR

**Comment by iLLiCiTiT** [comment]
Ready to merge but avalon's PR must be first https://github.com/pypeclub/avalon-core/pull/392 (and https://github.com/Ellipsanime/avalon-core/pull/1).

File: website/docs/admin_settings_project_anatomy.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ We have a few required anatomy templates for OpenPype to work properly, however
 | `task[name]` | Name of task |
 | `task[type]` | Type of task |
 | `task[short]` | Shortname of task |
+| `parent` | Name of hierarchical parent |
+| `task` | Name of task |
```
Comment (iLLiCiTiT) [inline website/docs/admin_settings_project_anatomy.md:64]: Task should not be here.
```suggestion
```

**Comment by iLLiCiTiT** [comment]
> Ready to merge but avalon's PR must be first pypeclub/avalon-core#392 (and Ellipsanime/avalon-core#1).

@ClementHector I just want to remind that this is ready but avalon-core PR must be merged first. And the PR is not ready to merge yet,

**Comment by ClementHector** [comment]
@iLLiCiTiT I think it's good now.

**Review by iLLiCiTiT — APPROVED** [review]

**Pull request merged**